csharp/AppEncryption/AppEncryption.Tests/AppEncryption/Persistence/DynamoDbMetastoreImplTest.cs

```diff
@@ -42,12 +43,18 @@ public class DynamoDbMetastoreImplTest : IClassFixture<DynamoDBContainerFixture>
 
         public DynamoDbMetastoreImplTest(DynamoDBContainerFixture dynamoDbContainerFixture)
         {
+            AmazonDynamoDBConfig clientConfig = new AmazonDynamoDBConfig
+            {
+                ServiceURL = dynamoDbContainerFixture.ServiceUrl,
+                AuthenticationRegion = "us-west-2",
+            };
+            amazonDynamoDbClient = new AmazonDynamoDBClient(clientConfig);
+
+            CreateTableSchema(amazonDynamoDbClient, "EncryptionKey");
+
             dynamoDbMetastoreImpl = NewBuilder(Region)
-                .WithEndPointConfiguration(dynamoDbContainerFixture.ServiceUrl, "us-west-2")
+                .WithEndPointConfiguration("http://localhost:8000", Region)
```

Comment:
Why not use `dynamoDbContainerFixture.ServiceUrl` here?

Reply:
fixed